.github/workflows/coverage.yml

```diff
@@ -21,7 +23,7 @@ jobs:
         with:
           submodules: true
       - name: Add safe directory
-        run: git config --global --add safe.directory /__w/envoy-mobile/envoy-mobile
+        run: git config --global --add safe.directory /__w/envoy/envoy
```

Comment:
I don't think these are needed anymore because there's no git submodule involved anymore, just the main git repo.

Reply:
CI broke without them, and told me to add them back.

Reply:
Odd.